widgets/shortvideo/inline-shortvideo-swiper.loader.ts

```diff
@@ -0,0 +1,153 @@
+import { Sdk, SwiperData } from "types"
```

Comment:
How is this different from carousel swiper initialization? If the only different is the id and navigation button parameters, should we somehow reuse the logic from carousel? Maybe we can work on this later in a separate PR

Reply:
It is the same as carousel swiper, we can abstract the swiper and reuse it. Let's do it in a separate PR